packages/phpstan-latte-rules/tests/Rules/LatteCompleteCheckRule/Source/link.latte

```diff
@@ -0,0 +1,5 @@
+<a href="{link doSomething!, 'a'}">OK link</a>
+<a n:href="doSomething!, 'b', ['c' => 'd']">OK n:href</a>
+<a href="{link doSomething!, 'e', 'f', 'g'}">wrong arguments</a>
+<a n:href="doSomething!, $foo, $bar">with variables</a>
+<a href="{link unknown!, 'x'}">wrong link</a>
```

Comment:
Thanks! Could you add test fixture with 2 links to 2 different target classes?

Reply:
Do you mean link to another presenter? No in this PR. This is just about handleX methods. 

For other links I need to solve several issues:
- how to get PresenterFactory with mapping
- how to process actionX / renderX / nothing

Reply:
I'm not sure how signals work in Nette, don't use them much.

Reply:
This is how I understand it and use it: you can create link to handle method (signal) only in the same class or template which belongs to it. You can't create cross component handle link.

Reply:
That should be safe then, thanks for the feature ☺️